### ISSUE TITLE
Forkify App Additional Features

### ISSUE THREAD

**Issue by chrismeier30** [UmarUFLIP/JavaScriptCourseJS#4]
### Description

1. Add 5 new features to the Forkify App Project created in Section 18. Before coding the features below, create a new issue ticket for each feature and write the requirements. Reference the Pig Game Issue ticket for how to write a new issue ticket.

    - Display number of pages between the pagination buttons 
    - Allow user to add more then 6 ingredients
    - Improve recipe ingredient input (i.e use multiple fields instead of using commas) 
    - Perform ingredient validation in view, before submitting the form
    - Allow user to edit their recipe ingredients

    
2. Pick 1 out of the 3 Advanced Features below. Before coding the advanced feature, 
create a new issue ticket. The issue ticket should contain Description, Flow Chart, and Requirements (Checkboxes). You can include other documentation such as screenshots of new User Interface mockups or any reference material used to build the advanced feature.

    - Adv. Feature 1:  Shopping list. Add a button to the recipe that will add the ingredients to a list. Users will use this feature on multiple recipes that contain similar ingredients. Sometimes users will even click the add ingredient button more than once on the same recipe. Therefore do not add duplicate ingredients to the shopping list. When a user is looking at a recipe, notate any ingredients that are already on that user's shopping list. Sometimes user's change their mind. Allow users to remove ingredients from their shopping list. Once the user is done adding to their shopping list, they should be able to view their list in a nice readable and printable format. 

    - Adv. Feature 2: Weekly meal planning. Allow users to assign recipes to the next 7 days. Users can assign up to 5 recipes per day. Allow users to remove recipes from any of the upcoming days. Allow users to move recipes from 1 day to another. Users should be able to view the days with the assign recipes on a weekly calendar. When looking at the calendar, users can click on the recipe and view the ingredients needed to make the meal.

    - Adv. Feature 3: Get nutrition data. When a user is looking at a recipe display the total calories. Use the [Spoonacular Food API](https://spoonacular.com/food-api) to obtain the nutrition data and display to the user.
